features/confirmModal/ConfirmModal.tsx

```diff
@@ -0,0 +1,85 @@
+import { CloseIcon } from '@/assets/icons/close';
+import { Button } from '@/shared/ui';
+import { Typography } from '@/shared/ui';
+import {
+  Dialog,
+  DialogClose,
+  DialogContent,
+  DialogOverlay,
+  DialogPortal,
+  DialogTitle,
+} from '@radix-ui/react-dialog';
+
+import style from './ConfirmModal.module.scss';
+
+type ConfirmModalProps = {
+  children: ReactNode;
+  className?: string;
+  openConfirmModal: boolean;
+  openParentModal: boolean;
+  setOpenConfirmModal: (openConfirmModal: boolean) => void;
+  setOpenParentModal: (open: boolean) => void;
+  showCloseButton: boolean;
+  title: string;
+};
+
+export const ConfirmModal = (props: ConfirmModalProps) => {
+  const {
+    children,
+    className,
+    openConfirmModal,
+    openParentModal,
+    setOpenConfirmModal,
```

Comment:
Redundant `openConfirmModal` and `openParentModal`.
You can use RadixPrimitive. Modal props such as `open` and `onOpenChange`, which will suffice.